crates/astria-sequencer/src/transaction/signed.rs

```diff
@@ -0,0 +1,127 @@
+use anyhow::{
+    bail,
+    Context as _,
+    Result,
+};
+use astria_proto::sequencer::v1::{
+    unsigned_transaction::Value::AccountsTransaction as ProtoAccountsTransaction,
+    SignedTransaction as ProtoSignedTransaction,
+};
+use async_trait::async_trait;
+use penumbra_storage::{
+    StateRead,
+    StateWrite,
+};
+use prost::Message as _;
+use tracing::instrument;
+
+use crate::{
+    accounts::{
+        transaction::Transaction as AccountsTransaction,
+        types::Address,
+    },
+    crypto::{
+        Signature,
+        VerificationKey,
+    },
+    transaction::{
+        unsigned::Transaction as UnsignedTransaction,
```

Comment:
I realize this naming came as a fix from the clippy lint. If you rename `unsigned::Transaction` to `unsigned::Unsigned` and reexport it under `transaction`, you can refer to it as `transaction::Unsigned` without the exra `transaction::unsigned::Transaction as UnsignedTransaction` dance.

Apologies for not having made this clear last week. :(